### ISSUE TITLE
Unable to build the solution using .net core 2.0

### ISSUE THREAD

**Issue by Vaidehi17** [simplcommerce/SimplCommerce#506]
Hi,

I need to use SimplCommerce solution using .net core 2.0

When i downloaded and run this solution, it was pointing to .net core 2.1. I have changed the framework to .net core 2.0, i was unable to build the solution. I was getting number of errors.

Can you please help me run this solution using .net core 2.0

**Comment by thiennn** [comment]
.net core can be side by side. Why can't you use 2.1? I strongly recommend you to use .net core 2.1. The only way to get SimplCommerce work on 2.0 is to checkout its old version

**Comment by Vaidehi17** [comment]
Can i get the link to download SimplCommerce solution version which will work on 2.0?

**Comment by thiennn** [comment]
https://github.com/simplcommerce/SimplCommerce/tree/90c6a8c619e8d0309a8fb3b006d1e994e18244c8/

**Comment by Vaidehi17** [comment]
Thank you!

When i run add migration to create database schema, i was getting couple of errors like add the attribute [NotMapped] to UserAddresses and DefaultShippingAddress 
in User model. I added the attribute.

Then i got the following error,
The child/dependent side could not be determined for the one-to-one relationship that was detected between 'User.DefaultBillingAddress' and 
'UserAddress.User'. To identify the child/dependent side of the relationship, configure the foreign key property. If these navigations should not be part 
of the same relationship configure them without specifying the inverse. See http://go.microsoft.com/fwlink/?LinkId=724062 for more details.

I modified EFConfigurationDbContext.cs -> OnModelCreating method and added following code,
modelBuilder.Entity<User>()
            .HasOne(a => a.DefaultBillingAddress)
            .WithOne(b => b.User)
            .HasForeignKey<UserAddress>(b => b.User); 
			
Now when i run the solution, i am getting error as 

InvalidOperationException: Cannot create a DbSet for 'Entity' because this type is not included in the model for the context.

![image](https://user-images.githubusercontent.com/41745354/43386261-c01d682c-9400-11e8-8868-a17974711746.png)

How to resolve?
